### ISSUE TITLE
Feature Request: Ability to embed sheet into another note

### ISSUE THREAD

**Issue by andywhitaker** [divamgupta/obsidian-spreadsheets#14]
Love the plugin! I wanted to suggest the feature to embed a sheet into another note. Would be very handy to be able to reference with a ![[spreadsheet.sheet]] syntax. Not sure if possible to edit within that frame but that would be even better if possible.

Either way it's a great plugin! Thanks for the work!

**Comment by mdMcAlister** [comment]
Yes, please! I'd like to use obsidian as an electronic lab notebook and inline tables (capable of calculation) are the only thing I'm missing.
